### ISSUE TITLE
Define Colors

### ISSUE THREAD

**Issue by dodekeract** [3YOURMIND/kotti#7]
`#FFFFFF` is used a lot, but never defined in kotti. I'd suggest adding a `$color-white: #fff` rule.

**Comment by ostoc** [comment]
First of all, white and black color are basic colors which used everywhere, i don't think they should be included in a color palettes IMHO. 
Secondly, for a clean and readable code, you can just use `white` or `black` in CSS

**Comment by dodekeract** [comment]
@ostoc Yes, but then that should be mentioned in the documentation. I also don't see any reason not to include them in our official color table. For example, I assumed that we'd just always use `$lightgray-300` (`#F8F8F8`), or `$darkgray-600` (`#3D3D3D`). There are people that never use "true" black, so I assumed that's intentional.

I mean, I'm fine if we don't require setting it in SCSS, but it should be mentioned in the color palette that we DO use these colors.

It's also not uncommon to define basic colors, just look at the [YouTube Colors section in their Brand Resources](https://www.youtube.com/yt/about/brand-resources/#logos-icons-colors). Remembering it does not make it obvious that you're encouraged to use it.

**Comment by igeligel** [comment]
Atlassian is also including `#FFFFFF`. See [here](https://atlassian.design/guidelines/brand/color-1). Also they never use black. That is why it is not listed there.

I also need some yellow colors now for the new 3D Viewer. Would be nice if you define them.
